components/style/properties/longhand/box.mako.rs

```diff
@@ -149,6 +149,7 @@ ${helpers.single_keyword("clear", "none left right both",
 <%helpers:longhand name="vertical-align"
                    animatable="True">
   use cssparser::ToCss;
+  use parser::Parse;
```

Comment:
You can put this trait in the imports in https://github.com/servo/servo/blob/master/components/style/properties/helpers.mako.rs#L176